DEVDOCS-5183: [update] add note about editing variants in control panel (#2090)
bigcommerce/dev-docs: docs/api-docs/catalog/products-overview.mdx

`https://api.bigcommerce.com/stores/{{store_hash}}/v3/catalog/products/{{product_id}}/variants`

<Callout type="info">

>   * If you want to prevent some option combinations from having a variant, do not edit the variants in the control panel, as that will cause the store to autogenerate all missing variant SKUs.

Reviewer: ```suggestion
  * If you want only some option combinations to have a variant, do not edit the variants in the control panel, as that will cause the store to autogenerate all missing variant SKUs.
```
